client/verta/tests/unit_tests/strategies.py

```diff
@@ -262,3 +260,98 @@ def mock_workspace(draw):
         )
     )
     return workspace
+
+
+@st.composite
+def mock_pipeline_definition(draw):
+    """Generate a mocked pipeline specification dictionary"""
+
+    # step names in a pipeline must be unique
+    step_names = draw(
+        st.lists(st.text(min_size=5, max_size=25), min_size=5, max_size=5, unique=True)
+    )
+    model_versions = draw(
+        st.lists(
+            st.text(alphabet=["1", "2", "3", "4", "5"], min_size=1),
```

Comment:
It looks like we could instead draw from `st.integers()`.

Reply:
Especially since [`model_version_id` is supposed to be an integer](https://github.com/VertaAI/services/blob/d2038ec88b1537f4286619b15bcaaa5c30f520fb/deployment/deployment-api/api/schema/definitions/PipelineDefinition.yaml#L21-L22).

Reply:
Using integers was leading to a protobuf parsing error, but I finally figured out it was because the mocked http responses needed actual json responses with the bare minimum fields expected.  It has now been updated to use integers and the tests are passing.